core/src/main/java/org/jboss/logmanager/LoggerNode.java

```diff
@@ -386,7 +389,22 @@ void publish(final ExtLogRecord record) {
         } catch (VirtualMachineError e) {
             throw e;
         } catch (Throwable t) {
-            // todo - error handler
+            ErrorManager errorManager = handler.getErrorManager();
```

Comment:
This should probably be retrieved in a privileged action.

Reply:
Fixed.